ocs_ci/utility/aws.py

```diff
@@ -262,50 +269,75 @@ def detach_and_delete_volume(self, volume, timeout=120):
             delete_response
         )
 
-    def stop_ec2_instance(self, instance_id, wait=False):
+    def stop_ec2_instances(self, instances, wait=False, force=True):
         """
         Stopping an instance
 
         Args:
-            instance_id (str): ID of the instance to stop
+            instances (dict): A dictionary of instance IDs and names to stop
             wait (bool): True in case wait for status is needed,
                 False otherwise
+            force (bool): True for force instance stop, False otherwise
 
-        Returns:
-            bool: True in case operation succeeded, False otherwise
         """
-        res = self.ec2_client.stop_instances(
-            InstanceIds=[instance_id], Force=True
-        )
+        instance_ids = [key for key in instances.keys()]
+        instance_names = [val for val in instances.values()]
```

Comment:
Would replace 283 and 284 with a more elegant solution - 
```suggestion
        instance_ids, instance_names = zip(*instances.items())
```